crates/wash-cli/src/up/mod.rs

```diff
@@ -537,7 +540,24 @@ pub async fn handle_up(cmd: UpCommand, output_kind: OutputKind) -> Result<Comman
     // Download wasmCloud if not already installed
     let wasmcloud_executable = if !wasmcloud_opts.start_only {
         spinner.update_spinner_message(" Downloading wasmCloud ...".to_string());
-        ensure_wasmcloud(&wasmcloud_opts.wasmcloud_version, &install_dir).await?
+        let new_wasmcloud_patch_version = new_patch_version_of_after_string(
```

Comment:
Another question @brooksmtownsend 
you said if someone is "specifying the version of wasmcloud". Currently there is only one string in the [WasmCloudOpts](https://github.com/wasmCloud/wasmCloud/blob/e2f6b61f7850c0dd51af1b9d582f44ab8ead633e/crates/wash-cli/src/up/mod.rs#L140-L141), which - if specified - will be overwritten.

So how should we know if someone used the `--wasmcloud-version`? How to prevent the 'patching' if a specific version was added 🤔 ?

Reply:
Sorry the questions was: can I mess up the current [WasmCloudOpts](https://github.com/wasmCloud/wasmCloud/blob/e2f6b61f7850c0dd51af1b9d582f44ab8ead633e/crates/wash-cli/src/up/mod.rs#L140-L141)?

Reply:
Yes, thank you. I added the `Option<String>` already for the necessary places, but there are some integration tests, which are failing. So it requires some massage still. Thanks for the detailed answer. 🎉